tests/system/test_system.py

```diff
@@ -46,7 +48,11 @@ def _get_credentials_and_project():
 
 @pytest.fixture(scope=u"module")
 def client():
-    credentials, project = _get_credentials_and_project()
+    if FIRESTORE_EMULATOR:
+        credentials = EMULATOR_CREDS
+        project = FIRESTORE_PROJECT
+    else:
+        credentials, project = _get_credentials_and_project()
```

Comment:
This logic needs to be inside `_get_credentials_and_project`, so that it is used by `test_collections_w_import` below.